mrmustard/physics/converters.py

```diff
@@ -60,7 +60,10 @@ def to_fock(rep: Representation, shape: Optional[Union[int, Iterable[int]]] = No
             msg += f"the number of variables of this ansatz ({rep.ansatz.num_vars})."
             raise ValueError(msg)
 
-        array = [math.hermite_renormalized(A, b, c, shape) for A, b, c in zip(rep.A, rep.b, rep.c)]
+        array = [
+            math.hermite_renormalized(A, b, complex(c), shape)
```

Comment:
`atleast_2D` has a `dtype` option. Could we just specify `dtype=complex` and avoid having to cast here?

Reply:
Making c at least 2d means c here is now an array, but `hermite_renomalized` only takes scalars for c. I think `c[0]` should be used instead. We will however also need to be careful once the shape of c is no longer `(batch,1)`, since one will have to do an inner product between c and the array in line 63)

Reply:
Right makes sense. Looks like `hermite_renormalized` will just need to be refactored to account for `!=(batch,1)` cs